rdr_service/offline/study_nph_ingest_awardee_data.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from typing import Dict, Any, Iterator, Optional
+from csv import DictReader
+
+from rdr_service.model.study_nph import Site
+from rdr_service.dao.study_nph_dao import NphSiteDao
+
+
+_logger = logging.getLogger("rdr_logger")
+
+
+def read_csv(filepath: str) -> Iterator[Dict[str, Any]]:
+    with open(filepath, "r", encoding="utf-8-sig") as csv_fp:
+        csv_dict_reader = DictReader(csv_fp)
+        for row in csv_dict_reader:
+            yield row
```

Comment:
This is a very general method, and I think we can move forward w/o and move the function, the csv iteration directly into the tool.